src/lib/assets/skeletons/SkeletonTable.tsx

```diff
@@ -27,15 +31,19 @@ export const SkeletonTable: React.FC<SkeletonTableProps> = ({
     times(numRows * numCols, () => {
       elements.push(
         <>
-          <Skeleton width={`${getRandomInt(35, 75)}%`}></Skeleton>
+          <Skeleton
+            height={rowHeight}
```

Comment:
My table rows are much larger than the default Skeleton height, so I want the ability to modify this as needed.